src/utils/address.ts

```diff
@@ -1,9 +1,40 @@
-// Turns '0xce10ce10ce10ce10ce10ce10ce10ce10ce10ce10'
-// into ['ce10','ce10','ce10','ce10','ce10','ce10','ce10','ce10','ce10','ce10']
+/**
+ * Reference files:
+ * https://github.com/celo-org/developer-tooling/blob/master/packages/sdk/base/src/address.ts
+ * https://github.com/celo-org/developer-tooling/blob/master/packages/sdk/utils/src/address.ts
+ */
+
+import { privateToPublic, toChecksumAddress } from '@ethereumjs/util'
```

Comment:
Can we try using the viem equivalent functions?
[getAddress](https://viem.sh/docs/utilities/getAddress#getaddress) and maybe [privateKeyToAccount](https://viem.sh/docs/accounts/local/privateKeyToAccount#privatekeytoaccount)?

Fine to do as a separate PR.

To avoid pulling `@ethereumjs/util`.